input.css

```diff
@@ -50,4 +50,24 @@
     p a {
         @apply underline;
     }
+
+    .shortcut-helper {
+        position: relative;
+    }
+
+    .helper-text {
+        display: none;
+        position: absolute;
+        border: 1px solid;
+        padding: 2px 6px;
+        font-size: 12px;
+        right: -15px;
+        margin-top: -5px;
+        background: pink;
+        color: black;
+    }
+
+    .show_modifiers .helper-text {
+        display: block;
+    }
```

Comment:
This needs cleaned up, but the style in general probably needs rethought, so it's fine for now.